config/locales/en.models.yml

```diff
@@ -330,6 +330,7 @@ en:
         canceled_by: Canceled By
         canceled_at: Canceled At
         type: Type
+        expires_mins_before: Expires Time Prior
```

Comment:
I'm not sure what, but I feel like we should try to come up with a better label for this field. Maybe a hint would help?

Reply:
We'll see what NU thinks.